cli/tests/test_release_preflight.py

```
@@ -0,0 +1,936 @@
+def test_future_matrix_fails_closed_when_exact_field_recovery_anchor_is_missing(
+    tmp_path: Path,
+) -> None:
+    policy = _baseline_policy()
+    policy["published_baselines"].remove("0.8.6")
+    policy["published_baseline_config_versions"].pop("0.8.6")
+    path = tmp_path / "effective.json"
+    path.write_text(json.dumps(policy), encoding="utf-8")
+
+    with pytest.raises(
+        release_preflight.ReleasePreflightError,
+        match="field-recovery baseline 0.8.6 is unavailable",
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)